build/package/nfpm.yaml

```diff
@@ -34,6 +34,7 @@ contents:
   type: tree
 - src: nfpm-prefix/etc/kong
   dst: /etc/kong
+  type: dir
```

Comment:
let's remove the `src` here, i.e. create an empty directory, since we copy the files later.

Reply:
done